tests/src/com.graphql-java/graphql-java-extended-validation/19.1/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    implementation "com.graphql-java:graphql-java-extended-validation:$libraryVersion"
+    implementation "org.jboss.logging:jboss-logging:3.5.0.Final"
+    implementation "org.hibernate.validator:hibernate-validator:7.0.4.Final"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+	binaries {
+		test {
+			buildArgs.add('--no-fallback')
+			buildArgs.add('-H:IncludeLocales=fr')
+		}
+	}
+}
```

Comment:
- You can change to the following block of code to verify ?
```groovy
graalvmNative {
    binaries {
        test {
            // We have to manually add the metadata because the org.graalvm.internal.tck gradle plugin does not automatically import
            configurationFileDirectories.from(tck.metadataRoot.dir('com.graphql-java/graphql-java/19.2'))
            configurationFileDirectories.from(tck.metadataRoot.dir('org.hibernate.validator/hibernate-validator/7.0.4.Final'))
        }
    }
}
```

Reply:
No need to, I've manually copied those folders in the test and it worked. I guess this approach is more elegant but doesn't solve the TCK problem.